src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -108,6 +108,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@PaulHigin, your last commit had 11 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

get-help other tests.get-help helpFunc11.$_.description
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Scripting/ScriptHelp.Tests.ps1: line 544
544:         It '$_.description' { $_.description[0].text | Should -Match "^$($_.Name)\s+help" }

```
get-help other tests.get-help helpFunc11.$_.description
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Scripting/ScriptHelp.Tests.ps1: line 544
544:         It '$_.description' { $_.description[0].text | Should -Match "^$($_.Name)\s+help" }

```
get-help other tests.get-help helpFunc11.$_.description
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Scripting/ScriptHelp.Tests.ps1: line 544
544:         It '$_.description' { $_.description[0].text | Should -Match "^$($_.Name)\s+help" }

```
get-help other tests.get-help helpFunc11.$_.description
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Scripting/ScriptHelp.Tests.ps1: line 544
544:         It '$_.description' { $_.description[0].text | Should -Match "^$($_.Name)\s+help" }

```
get-help other tests.get-help helpFunc11.$_.description
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Scripting/ScriptHelp.Tests.ps1: line 544
544:         It '$_.description' { $_.description[0].text | Should -Match "^$($_.Name)\s+help" }

```